### PR TITLE
Age text to welcome screen

### PR DESCRIPTION
**Pull request by Osamah-learn**
Finnish
Screenshot 2021-09-29 at 13 51 36
English
Screenshot 2021-09-29 at 14 00 31

### CODE REVIEW

File: src/Screens/Onboarding/Welcome.tsx [hunk 1]
```diff
@@ -42,7 +43,16 @@ export default ({ navigation }: Props) => {
...
           <Message style={styles.text} id={'onboarding.welcome.text2'} />
-          <Message style={styles.text} id={'onboarding.welcome.text3'} />
+          <Message
+            style={styles.textWithoutMargin}
+            id={'onboarding.welcome.text3'}
+          />
+          <Link
+            style={styles.link}
+            linkName="onboarding.welcome.apuu.link"
+            url={'https://apuu.fi/'}
```
Comment (rottabonus) [inline src/Screens/Onboarding/Welcome.tsx:53]: This URL could be introduced similarly like rest of the urls

**Review by rottabonus — COMMENTED** [review]
Great, I would like that the apuu-chat-url is in style with other urls in the repo

**Review by ahto — APPROVED** [review]
I was doing the changes with Osku so they look good to me 😏

**Review by idolgov — APPROVED** [review]

**Comment by rottabonus** [comment]
This PR https://github.com/sos-lapsikyla/ylitse-app/pull/177 contains these changes. Closing this one, and merging #177

**Pull request closed without merging**